include/rocksdb/merge_operator.h

```diff
@@ -183,6 +183,13 @@ class MergeOperator {
   //       no checking is enforced. Client is responsible for providing
   //       consistent MergeOperator between DB opens.
   virtual const char* Name() const = 0;
+
+  // Determines whether the MergeOperator can be called with just a single
+  // merge operand.
+  // Override and return true for allowing single operands. FullMergeV2 and
+  // PartialMerge/PartialMergeMulti should be implmented accordingly to handle
+  // a single operand.
+  virtual bool DoesAllowSingleMergeOperand() const { return false; }
```

Comment:
I was just trying to follow some convention of using Is/Has/Does/etc prefix for boolean functions. Since there is no strict written convention anywhere, I don't have a strong preference either way and I can drop the Does prefix.